### PR TITLE
Initial version of Dockerfile

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: Dockerfile
Comment (tmooney) [inline Dockerfile:7]: Should we combine this `apt-get update` with the following `apt-get install` (or do a second one)?

I suppose the "efficient" thing to do would be to combine all our `apt-get install`s in this file and combine them into one.  But this is certainly easier for development!

Comment (susannasiebert) [reply Dockerfile:7]: Yeah, I wasn't sure about that either. It's kinda nice to have them separate to know which software has which dependencies. But it's certainly more efficient to combine and deduplicate these.

File: Dockerfile
Comment (jasonwalker80) [inline Dockerfile]: Do we need to add pindel2vcf : `RUN ln -s /opt/pindel-0.2.5b8/pindel2vcf /usr/bin/pindel2vcf`

**Review by jasonwalker80 — CHANGES_REQUESTED** [review]

File: Dockerfile
Comment (jasonwalker80) [reply Dockerfile]: Also might need to `COPY somatic_indelfilter.pl /usr/bin/somatic_indelfilter.pl`

File: Dockerfile
Comment (jasonwalker80) [inline Dockerfile]: Should this be `--prefix=$HTSLIB_INSTALL_DIR` instead

File: Dockerfile
Comment (jasonwalker80) [inline Dockerfile]: If the above change is made, this would be `--with-htslib=$SAMTOOLS_INSTALL_DIR/htslib-1.3.2`

File: Dockerfile
Comment (jasonwalker80) [inline Dockerfile]: If the above changes are made to the htslib prefix, this would be `$SAMTOOLS_INSTALL_DIR/htslib-1.3.2` this change might get us passed the pindel Makefile issue.

**Comment by jasonwalker80** [comment]
Comments removed about changing the install path.  Modifying the pindel Makefile seems to do the trick.

File: Dockerfile
Comment (tmooney) [inline Dockerfile:84]: After we `make` htslib, we should probably `rm` this temporary directory.

(Optionally, it could first be used to install pindel first in lieu of running `sed`.  The case where the original download directory is used is not broken in the pindel `Makefile`.)

**Comment by susannasiebert** [comment]
I made all of the requested changes and the Dockerfile builds successfully on my laptop. I will merge once it's approved.

**Review by tmooney — APPROVED** [review]
+1 :small_airplane:

**Review by jasonwalker80 — APPROVED** [review]
+1

**Pull request merged**
